test/Makefile.am

```diff
@@ -70,18 +70,21 @@ test_va_api_SOURCES =						\
 test_va_api_LDFLAGS =						\
 	$(LIBVA_LDFLAGS)					\
 	$(LIBVA_DRM_LDFLAGS)					\
+	$(DRM_LDFLAGS)						\
```

Comment:
Instead of adding these, you can:
 - update configure.ac to check for both `PKG_CHECK_MODULES([LIBVA_DRM], [libva-drm libdrm]`
 - drop any existing `DRM_LIBS` and `DRM_CFLAG` references
 
 After all, one cannot have libva-drm without libdrm.

Reply:
Done in the latest, I think